src/main/scala/dsptools/Chisel3TestersCompatibility.scala

```diff
@@ -0,0 +1,14 @@
+package chisel3.iotesters
```

Comment:
Ok, I double checked. Pretty much all of the functions here are so that I can override a method that's like 90% identical to the old one with minor tweaks. (flatten, bigIntToStr, validName). validName was also used for dspPeek/Poke logger printing. 

I wanted getDataNames for being able to print a Verilog TB file just like how VerilatorBackend sets stuff up.